### DISCUSSION TITLE
apache lisence

### DISCUSSION THREAD

**Discussion by free2ride19** [community/community#27707] [Apps, API and Webhooks]
I need a link for apache license if available for free or mainly for project.
@free2ride19

**Comment by airtower-luna** [comment]
You can get the license file right from the ASF: https://www.apache.org/licenses/LICENSE-2.0

You can use code under the Apache License for free, as long as you follow the conditions defined in the license. [Choose a License has a summary](https://choosealicense.com/licenses/apache-2.0/) of what the conditions and permissions are. As always: Consult a lawyer if you need legal advice.

**Comment by free2ride19** [comment]
https://form.jotform.com/220476043605046
Thank you for the information, my survey response link for more update.

**Comment by fernandounger** [comment]
### You can put the Apache license in your project by following these steps

- In your project repository go to add file and create a new one

![image](https://user-images.githubusercontent.com/87918106/181919483-9989e168-a212-401f-b7da-239679f397f9.png)
![image](https://user-images.githubusercontent.com/87918106/181919488-fe2f5530-0116-44ef-8e5d-c259d11749c5.png)

- Type ```LICENSE``` and click ```Choose a license template```

![image](https://user-images.githubusercontent.com/87918106/181919507-cb9d530b-79f8-40da-9f98-c7b7f33c2811.png)

- Done, on the left you can add the apache license

![image](https://user-images.githubusercontent.com/87918106/181919524-ccaf5642-b040-4d30-acb5-fb9f2fd839ed.png)

- Selected, just click on ```review and submit```

![image](https://user-images.githubusercontent.com/87918106/181919860-9f07cc6d-62b3-4749-aec1-abf15f89ebb2.png)



> I need a link for apache license if available for free or mainly for project. @free2ride19

**Comment by free2ride19** [comment]
Thank you for the feedback
